docs/production-on-sapir.md

```diff
@@ -1,13 +1,16 @@
 Production on Sapir
 -------------------
 
-As of <time datetime="2019-11-18">November 18, 2019</time>, the Cree
-Dictionary is deployed on a server called **Sapir**. You can visit the
+As of <time datetime="2021-01-05">January 5, 2021</time>, the itwêwina
```

Comment:
Is `the itwêwina` correct? I’d been thinking of the project name as a proper noun.

Reply:
Good catch!